### DISCUSSION TITLE
More funds were locked, than the channel itself after force close.

### DISCUSSION THREAD

**Discussion by ybaidiuk** [lightningnetwork/lnd#8003] [Q&A]
I did force close but still can not figure out some logic.
https://mempool.space/address/bc1qnj29s89v67pa4cepffcalx8kcy5kzgrdhnpqvrryh895a42gefts5d87cm

Did chart for better visualization
![image](https://github.com/lightningnetwork/lnd/assets/11969255/b1dca490-f8ca-45cd-99c8-43255deb0005)
![image](https://github.com/lightningnetwork/lnd/assets/11969255/4f2da5da-db37-4cb9-9a5e-06da5bb48e1f)
have 3 question for advanced developers, because I can not figure it out 2 days already.

1. Why was touched my sats (...ccl3 address), which was not attached to this channel at all? now because of 20k sats i get locked 500k for 2 weeks...
2. Why lock time 539184385 point on 1987 year?
3. How I can claim later sats from ...mptr address?

Please help me figure out if it okay or some potential bug with LND.

**Comment by guggero** [comment]
You can read about anchor output channels here: https://bitcoinops.org/en/topics/anchor-outputs/

The reason more funds were added was to pay the fees for getting the force-close transaction confirmed. That's what the 330 sat output is for, it's called an anchor. So the ...ccl3 output was added to pay the fees for it.
But those funds will not be locked for two weeks, only until the two transactions are confirmed.

> Why lock time 539184385 point on 1987 year?

That's actually an interesting question. I guess it doesn't matter much as long as the lock time is in the past (otherwise the transaction couldn't be accepted into the mempool). But no idea where that value comes from. Did you open the channel or was it opened to your node? My guess would be that it was opened by the other party that uses different values than `lnd`.

> How I can claim later sats from ...mptr address?

You don't need to do anything other than letting the node run, it will do it automatically after the time lock (CSV) has expired.

**Comment by ybaidiuk** [comment]
> You can read about anchor output channels here: https://bitcoinops.org/en/topics/anchor-outputs/
> 
> The reason more funds were added was to pay the fees for getting the force-close transaction confirmed. That's what the 330 sat output is for, it's called an anchor. So the ...ccl3 output was added to pay the fees for it. But those funds will not be locked for two weeks, only until the two transactions are confirmed.
> 
> > Why lock time 539184385 point on 1987 year?
> 
> That's actually an interesting question. I guess it doesn't matter much as long as the lock time is in the past (otherwise the transaction couldn't be accepted into the mempool). But no idea where that value comes from. Did you open the channel or was it opened to your node? My guess would be that it was opened by the other party that uses different values than `lnd`.
> 
> > How I can claim later sats from ...mptr address?
> 
> You don't need to do anything other than letting the node run, it will do it automatically after the time lock (CSV) has expired.

I read bitcoinops article before, it said nothing about locking additional sats. 
but thanks for answer anyway)
